runtime/src/proofs.rs

```diff
@@ -1,5 +1,6 @@
 use codec::{Decode, Encode};
 use sp_core::H256;
+use sp_runtime::RuntimeDebug;
```

Comment:
why do we need this ?

Reply:
It is from merge of old version code in master. I will remove it.